common/motion_utils/include/motion_utils/resample/resample_utils.hpp

```diff
@@ -0,0 +1,117 @@
+// Copyright 2022 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef MOTION_UTILS__RESAMPLE__RESAMPLE_UTILS_HPP_
+#define MOTION_UTILS__RESAMPLE__RESAMPLE_UTILS_HPP_
+
+#include <motion_utils/constants.hpp>
+#include <motion_utils/trajectory/trajectory.hpp>
+#include <tier4_autoware_utils/geometry/geometry.hpp>
+#include <tier4_autoware_utils/geometry/path_with_lane_id_geometry.hpp>
+
+#include <vector>
+
+namespace resample_utils
+{
+constexpr double CLOSE_S_THRESHOLD = 1e-6;
```

Comment:
@purewater0901 
please make a common variable for this in threshold.hpp

Reply:
at another PR

Reply:
Since this value will not be common in other packages, I think it is unnecessary to move it to the common variables.